### ISSUE TITLE
"(Gekijouban) Shakugan no Shana" specials mapping

### ISSUE THREAD

**Issue by atraholt** [ScudLee/anime-lists#150]
This one's a bit of a trickster.

TheTVDB maps The Sovereign Carmel-san, s0e05.
https://www.thetvdb.com/series/shakugan-no-shana/seasons/0
AniDB considers this a T for the movie, Gekijouban Shakugan no Shana. and maps it as T5 **and** T6. In all technicality, these two things aired back to back with a commercial break between them.
https://anidb.net/perl-bin/animedb.pl?show=anime&aid=4388

I'm not sure Ts can be mapped?

**Comment by ScudLee** [comment]
In the Kodi scraper, Ts are treated as specials starting at 201+, so it is possible to map them.  Other apps might not recognize that though.  
The best you can do in this case is map the multiple anidb episodes to the same tvdb episode, so they'll all have the same tvdb plot and thumbnail.

My read on the tvdb [plot summary](https://www.thetvdb.com/series/shakugan-no-shana/episodes/375729), though, suggests that it also includes T4 ("Shakugan no Shana The Movie Promotional Video").  Do you agree?

**Comment by ScudLee** [comment]
Having not got any of the files to check (or even watched Shana past the first season), I'll defer to your judgement on T4.

As for HAMA, you'll have to ask over there how they handle trailer mappings.  (There are, by my count, two other instances already in the mapping list.)

The full fix will be: 
```xml
<anime anidbid="4388" tvdbid="78962" defaulttvdbseason="0" episodeoffset="3" imdbid="tt0994794">
    <name>Gekijouban Shakugan no Shana</name>
    <mapping-list>
      <mapping anidbseason="0" tvdbseason="0">;1-6;2-0;3-0;4-0;6-0;7-0;205-5;206-5;</mapping>
    </mapping-list>
  </anime>
```

**Comment by atraholt** [comment]
Also worth mentioning that importing the equivalent special from MyAnimeList also only marked T5 and T6 on AniDB, so I think we've got it right.

**Comment by ScudLee** [comment]
Fixed in 43f1b6d40e0913e3040d3a5ddc0ecbcbe90c7e26 (misnamed the commit)
